async_upnp_client/client_factory.py

```diff
@@ -143,7 +143,14 @@ async def _async_create_service(
         """Retrieve the SCPD for a service and create a UpnpService from it."""
         scpd_url = service_description_el.findtext("device:SCPDURL", None, NS)
         scpd_url = urllib.parse.urljoin(base_url, scpd_url)
-        scpd_el = await self._async_get(scpd_url)
+
+        try:
+            scpd_el = await self._async_get(scpd_url)
+        except UpnpXmlParseError:
+            if not self._non_strict:
+                raise UpnpXmlContentError(f"Ignoring bad XML document from URL {scpd_url}")
```

Comment:
```
        except UpnpXmlParseError as exception:
            if not self._non_strict:
                raise UpnpXmlContentError(f"Ignoring bad XML document from URL {scpd_url}") from exception
```

Reply:
Ok, great Steven! I should have fixed it as in your suggestion.
Should be ok now.